docs/conf.py

```diff
@@ -0,0 +1,17 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+import os
+from rocm_docs import ROCmDocs
+
+docs_core = ROCmDocs("MIOpen Documentation")
+docs_core.run_doxygen()
```

Comment:
If you are running doxygen here then you should remove the doxygen run from the cmake.

Reply:
Removed cmake doc files in [de9b364](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2053/commits/de9b364d58ccbc9968a9fe79cbbcc4f763e873ed)

Reply:
Updated docs/CMakelists.txt to use rocm cmake [a48e0d3](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2053/commits/a48e0d3024988a8e018521aca397919b7d461274)